scripts/utils/trading_strategy_helpers.js

```diff
@@ -429,30 +429,35 @@ withdrawal of or to withdraw the desired funds
   const buildTransferApproveDepositFromOrders = async function(
     masterAddress,
     bracketAddresses,
+    targetTokenAddress,
     stableTokenAddress,
+    lowestLimit,
+    highestLimit,
+    targetPrice,
     investmentStableToken,
-    targetTokenAddress,
     investmentTargetToken,
     storeDepositsAsFile = false
   ) {
     const fleetSize = bracketAddresses.length
-    assert(fleetSize % 2 == 0, "Fleet size must be a even number")
+    const stepSizeAsMultiplier = Math.pow(highestLimit / lowestLimit, 1 / bracketAddresses.length)
+    const bracketIndexAtTargetPrice = Math.floor(Math.log(targetPrice / lowestLimit) / Math.log(stepSizeAsMultiplier))
```

Comment:
`Math.round` is probably safer than `Math.floor`, so if the result of the division is 11.99999999 we return 12 instead of 11.